installing/installing_openstack/installing-openstack-installer-ovs-dpdk.adoc

```diff
@@ -0,0 +1,93 @@
+
+* Have a storage service installed in {rh-openstack}, like block storage (Cinder) or object storage (Swift). Object storage is the recommended storage technology for {product-title} registry cluster deployment. For more information, see xref:../../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage].
+
+* Have the metadata service enabled in {rh-openstack}.
+
+include::modules/installation-osp-default-deployment.adoc[leveloffset=+1]
+include::modules/installation-osp-control-compute-machines.adoc[leveloffset=+2]
+include::modules/installation-osp-bootstrap-machine.adoc[leveloffset=+2]
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+// TODO: New stuff. Should these actually come after the install-config.yaml is generated?
+include::modules/installation-osp-dpdk-preparation.adoc[leveloffset=+1]
+include::modules/installation-osp-dpdk-selecting-flavors.adoc[leveloffset=+2]
+include::modules/installation-osp-dpdk-creating-aggregate-groups.adoc[leveloffset=+2]
+include::modules/installation-osp-dpdk-enabling-multiqueue.adoc[leveloffset=+2]
+include::modules/installation-osp-dpdk-creating-additional-network-poll-driver.adoc[leveloffset=+2]
+
+include::modules/installation-osp-enabling-swift.adoc[leveloffset=+1]
+include::modules/installation-osp-verifying-external-network.adoc[leveloffset=+1]
+include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+include::modules/installation-initializing.adoc[leveloffset=+1]
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
+include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-deploying-bare-metal-machines.adoc[leveloffset=+2]
+include::modules/installation-osp-config-yaml.adoc[leveloffset=+2]
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+include::modules/installation-osp-accessing-api.adoc[leveloffset=+1]
+include::modules/installation-osp-accessing-api-floating.adoc[leveloffset=+2]
+include::modules/installation-osp-accessing-api-no-floating.adoc[leveloffset=+2]
+include::modules/installation-osp-configuring-sr-iov.adoc[leveloffset=+1]
+include::modules/installation-launching-installer.adoc[leveloffset=+1]
+include::modules/installation-osp-verifying-cluster-status.adoc[leveloffset=+1]
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+The cluster is operational. Before you can add OVS-DPDK compute machines, though, you must perform additional tasks.
+
+// TODO Duplicate required mods/language for DPDK and cut what's not necessary
+
+// include::modules/networking-osp-preparing-for-sr-iov.adoc[leveloffset=+1]
+// TODO: Metadata required or not?
```

Comment:
config-drive or metadata are required. Note that all Red Hat OpenStack clouds provide metadata service, so no worries.

Reply:
```suggestion
```